javascript/deprecate.js

```diff
@@ -0,0 +1,19 @@
+let deprecationWarnings = false

```

Comment:
I would just set this to `true` by default so the users can actually see them

Reply:
Oh, right. That will mean that I should reconsider how line 507 works, too.